src/repair/sources.rs

```diff
@@ -122,32 +125,77 @@ fn insert_entry<P: AsRef<Path>>(
 }
 
 pub fn create_new_sources_list(release: &str) -> io::Result<()> {
```

Comment:
You can change `-> io::Result<()>` to `-> anyhow::Result<()>` for those errors below.